src/ct.rs

```diff
@@ -93,15 +90,14 @@ where
             return CtDecryptResult(Choice::from(0));
         }
 
+        let (ctr, mask) = self.init_ctr(nonce);
```

Comment:
to mimic the change at https://github.com/mobilecoinfoundation/oblivious-aes-gcm/pull/5/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R274